File: utilities/src/main/java/com/toocol/ssh/utilities/execeptions/RemoteDisconnectException.java
package com.toocol.ssh.utilities.execeptions;

/**
 * @author ZhaoZhe (joezane.cn@gmail.com)
 * @date 2022/4/19 16:49
 */
public class RemoteDisconnectException extends RuntimeException {

    public RemoteDisconnectException(String message) {
        super(message);
    }

}

File: utilities/src/main/java/com/toocol/termio/utilities/execeptions/RemoteDisconnectException.kt
package com.toocol.termio.utilities.execeptions

import java.lang.RuntimeException

/**
 * @author ZhaoZhe (joezane.cn@gmail.com)
 * @date 2022/4/19 16:49
 */
class RemoteDisconnectException(message: String?) : RuntimeException(message)